### ISSUE TITLE
Integrate Extended Emotions with `Toon` shader

### ISSUE THREAD

**Issue by mikhail-dcl** [decentraland/unity-explorer#443]
Well, I see 2 possible ways assigning Toon shader to the emotes' gameobjects:

- We can assign the new material directly and map all the properties just like we do for Avatars. In main [this logic is quite separated](https://github.com/decentraland/unity-explorer/blob/main/Explorer/Assets/DCL/AvatarRendering/AvatarShape/ComputeShader/AvatarMaterialConfiguration.cs), so with some small additional effort it will be possible
   - The shader can work without any custom skinning
   - Though it means no performance optimization will be provided
- Integrate it fully with the our custom skinning systems
   - Currently, it's not separated from the construction of Avatars themselves
   - Though the components themselves (despite they belong to the Avatar domain) are quite isolated already so it's the matter of reshuffling systems and a bit of components here and there
   - Ultimately we will have a group of systems that just handle custom skinning and they do not depend on the avatar component itself.

It will be a leftover once `emotes` branch is merged into `main`.
It's requested by @olavra for the visual fidelity
